build.gradle

```diff
@@ -562,6 +562,7 @@ project(':clients') {
     compile libs.lz4
     compile libs.snappy
     compile libs.slf4jApi
+    compile libs.jacksonDatabind
```

Comment:
Yeah, it seems to me like introducing a `kafka-serialization-json.jar` is the simplest option at this stage. And yes, it seems like a simple KIP may be the way forward.